examples/test_examples.py

```diff
@@ -114,7 +114,9 @@ def test_run_pl_glue(self):
             --max_seq_length=128
             """.split()
         if torch.cuda.is_available():
-            testargs += ["--fp16", "--gpus=1"]
+            testargs += ["--gpus=1"]
+        if is_cuda_and_apex_avaliable():
```

Comment:
Typo here I think?

Reply:
I only want fp16 if apex available and cuda available. Am I missing something?

Reply:
```suggestion
        if is_cuda_and_apex_available():
```
(GitHub didn't let me make a suggestion earlier for some reason.

Reply:
incredible catch!